pkg/hubble/metrics/flow/handler.go

```diff
@@ -56,10 +57,8 @@ func (h *flowHandler) Status() string {
 
 func (h *flowHandler) ProcessFlow(ctx context.Context, flow *flowpb.Flow) {
 	var typeName, subType string
-	switch flow.GetEventType().Type {
-	case monitorAPI.MessageTypeAgent:
-		typeName = "Agent"
-		subType = monitorAPI.AgentNotifications[monitorAPI.AgentNotification(flow.GetEventType().SubType)]
```

Comment:
This was just added two days ago. Just making sure we're now removing it intentionally since this is not any kind of "legacy cleanup".